samples/suit/flash_companion/README.rst

```diff
@@ -0,0 +1,119 @@
+.. _suit_flash_companion:
+
+SUIT flash companion
+####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The flash companion application allows the secure domain firmware to access the external memory during the :ref:`Software Updates for Internet of Things (SUIT) <ug_nrf54h20_suit_dfu>` firmware upgrade.
+
+Application overview
+********************
+
+The flash companion implements a device driver for the external flash device and provides an IPC service to the secure domain.
+The secure domain firmware can request from the flash companion to read, erase, or write data to the external memory.
+
+The application is meant to be booted by the secure domain while performing the firmware update process using the :ref:`SUIT <ug_nrf54h20_suit_dfu>` firmware upgrade.
+
+The flash companion firmware is not a stand-alone application.
+The application must be built as a child image of a different sample as it inherits the device tree configuration from the parent build.
```

Comment:
I suggest removing the "Related samples" subheading, and instead modify this section by adding:

```suggestion
The flash companion firmware is not a stand-alone application, it is intended to be used with the ``nrf54h_suit_sample`` to complete a firmware transfer with external flash. 
The application must be built as a child image of a different sample as it inherits the devicetree configuration from the parent build.
```

Reply:
Done and removed Related samples.